website/project/model.py

```diff
@@ -635,6 +635,11 @@ class Node(GuidStoredObject, AddonModelMixin, IdentifierMixin):
 
     creator = fields.ForeignField('user', backref='created')
     contributors = fields.ForeignField('user', list=True, backref='contributed')
+    # Time of last sent notification email to newly added contributors
+    # {<User._id>:
+    #   {'last_sent': time.time()}
+    # }
+    contributor_record = fields.DictionaryField(default={})
```

Comment:
Don't pass an empty dict as the default argument for a field, as it may be possible for that dict to be shared across objects. I believe you can pass a callable as a the default, so you could do `default=dict`.